packages/commons-ui/.lintstagedrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require("eslint-config-commons-ui/.lintstagedrc.js");
```

Comment:
Why the two eslint commons config [packages](https://github.com/CodeForAfrica/ui/tree/main/packages/eslint-config-commons-ui)?

Reply:
@gertie-sheshe Um, not sure what you mean...is it this https://github.com/CodeForAfrica/ui/tree/main/packages/eslint-config-commons-ui and https://github.com/CodeForAfrica/ui/tree/main/packages/config